src/rust/lib/graph-editor/src/component/node/port/output.rs

```diff
@@ -0,0 +1,311 @@
+//! Implements the segmented output port area.
+use crate::prelude::*;
+
+use enso_frp as frp;
+use enso_frp;
+use ensogl::data::color;
+use ensogl::display::Attribute;
+use ensogl::display::Buffer;
+use ensogl::display::Sprite;
+use ensogl::display::scene::Scene;
+use ensogl::display::shape::*;
```

Comment:
This is a very wide import. Let's use `*` imports only for prelude, traits, and local modules.